src/IssueViz.Security/IssueVizSecurityPackage.cs

```diff
@@ -58,6 +69,11 @@ protected override async Task InitializeAsync(CancellationToken cancellationToke
             await ShowToolWindowCommand.CreateAsync(this,
                 new CommandID(Constants.CommandSetGuid, Constants.TaintToolWindowCommandId),
                 TaintToolWindow.ToolWindowId);
+
+            bindingMonitor = componentModel.GetService<ITaintIssuesBindingMonitor>();
```

Comment:
I think the doc comment on `ITaintIssuesBindingMonitor` should make it clear that it doesn't do an initial sync; it's not obvious, and it's important from an implementation point of view.